Gemfile

```diff
@@ -10,7 +10,7 @@ gem 'sinatra',         '~> 1.4.2'
 gem 'rake',            '~> 0.9.2.2'
 gem 'multi_json'
 
-gem 'sentry-raven',    github: 'getsentry/raven-ruby'
+gem 'sentry-raven',    git: 'https://github.com/getsentry/raven-ruby'
```

Comment:
I think we should either switch to the one on RubyGems or at least use the commits for the released versions instead of a commit in between versions.